api.md

```diff
@@ -1229,6 +1230,39 @@ For a table with more than one space dimension:
 SELECT set_number_partitions('conditions', 2, 'device_id');
 ```
 
+
+---
+
+## set_replication_factor() [](set_replication_factor)
+Sets the replication factor of a distributed hypertable to the given value. 
+Changing the replication factor does not affect the number of replicas for existing chunks.
+Chunks created after changing the replication factor will be replicated 
+in accordance with new value of the replication factor.
+
+If existing chunks have less replicas than new value of the replication factor,
+the function will print a warning.
+
+#### Required Arguments [](set_replication_factor-required-arguments)
+
+|Name|Description|
+|---|---|
+| `main_table` | Identifier of the distributed hypertable to update the replication factor for.|
+| `replication_factor` | The new value of the replication factor. Must be greater than 0.|
```

Comment:
Put the data type in parentheses before the description. I.e. 
> (INTEGER) The new value...

Reply:
Also, note any default values

Reply:
> Put the data type in parentheses before the description. I.e.
> 
> > (INTEGER) The new value...

Done